src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -1478,10 +1478,22 @@ crucible_alloc_with_mutability_and_size mut sz bic opts lty =
            pure sz_
          Nothing -> pure (Crucible.toBytes memTySize)
 
+     alignment' <-
+       case alignment of
+         Just a -> do
+           when (a < memTyAlign) $ fail $ unlines
+             [ "User error: manually-specified alignment was less than needed"
+             , "Needed for this type: " ++ show memTyAlign
```

Comment:
maybe show both `memTy` and `memTyAlign`

Reply:
good idea

Reply:
See revision 950f106ba8fdc393019819d7084e44d99f2ee9af